server/utils/outOfServiceBedUtils.test.ts

```diff
@@ -85,7 +85,7 @@ describe('outOfServiceBedUtils', () => {
           text: 'Room',
         },
         {
-          text: 'Out of service from',
+          text: 'Start date',
```

Comment:
Do we want to change the user facing description? 
To me 'Out of service from' is clearer than 'Start date' but I'm not sure what discussions have happened around this whilst I've been away

Reply:
as discussed on slack, we will leave as is pending feedback